pkg/cloud/ibmcloud/actuators/cluster/actuator.go

```diff
@@ -17,38 +17,70 @@ limitations under the License.
 package cluster
 
 import (
+	"context"
+	"fmt"
+	"github.com/pkg/errors"
 	"k8s.io/klog"
+
+	providerv1 "sigs.k8s.io/cluster-api-provider-ibmcloud/pkg/apis/ibmcloud/v1alpha1"
+	"sigs.k8s.io/cluster-api-provider-ibmcloud/pkg/cloud/ibmcloud"
 	clusterv1 "sigs.k8s.io/cluster-api/pkg/apis/cluster/v1alpha1"
-	client "sigs.k8s.io/cluster-api/pkg/client/clientset_generated/clientset/typed/cluster/v1alpha1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-// Actuator is responsible for performing cluster reconciliation
-type Actuator struct {
-	clustersGetter client.ClustersGetter
-}
-
-// ActuatorParams holds parameter information for Actuator
-type ActuatorParams struct {
-	ClustersGetter client.ClustersGetter
+type IBMCloudClient struct {
+	params ibmcloud.ActuatorParams
+	client client.Client
 }
 
 // NewActuator creates a new Actuator
-func NewActuator(params ActuatorParams) (*Actuator, error) {
-	return &Actuator{
-		clustersGetter: params.ClustersGetter,
+// NewActuator creates a new Actuator
+func NewActuator(params ibmcloud.ActuatorParams) (*IBMCloudClient, error) {
```

Comment:
I'm not sure if we need to change the return type name. We try to `NewActuator` but we actually new a `IBMCloudCLient`. 

What about kepping `Actuator` here, it is generated name and without any issue I guess?

Reply:
it's identical to the machine actuator...

Reply:
@jichenjc 
It is not functional related. Keep it consistent now can be fine.